services/browser-service/src/browser_service/api.py

```
@@ -63,6 +83,125 @@ async def navigate(request: BrowserRequest, app_request: Request) -> BrowserResp
+
+    Returns:
+        Success message
+
+    Raises:
+        HTTPException: If adding cookies fails
+    """
+    browser_manager = app_request.app.state.browser_manager
+
+    try:
+        await browser_manager.add_cookies(session_id, cookies, url)
+        return {"message": "Cookies added successfully"}
```

Ruff (0.14.0)

Consider moving this statement to an `else` block

(TRY300)